repository/sparql_reader.go

```diff
@@ -23,6 +27,20 @@ type SparqlRepository interface {
 	Query(q string) (*sparql.Results, error)
 }
 
+type RetryingSparqlRepository struct {
+	repo *sparql.Repo
+}
+
+func (r *RetryingSparqlRepository) Query(q string) (*sparql.Results, error) {
+	results, err := r.repo.Query(q)
+	for i := 0; i < retries && err != nil && strings.Contains(err.Error(), "no such host"); i ++ {
+		log.Printf("Retrying repo, time %d", i+1)
```

Comment:
Retrying network connections is a fine practice, but what is the specific use case here?

Reply:
OK, I see your explanation in sul-dlss-labs/rialto-dev#1 - but what do you think about adding a little more time to wait, so we can cut down on the number of retries? Just worried about what happens if we have connection problems in production...

Reply:
Forget what got me to the 180 number, but will reduce and see what happens. Would prefer to keep the wait small so that it works faster in dev once network is hooked up.

Reply:
Please add a comment to briefly explain why this is necessary, and then we're good to go.